src/main/java/seedu/address/ui/PersonDetailedViewContentManager.java

```diff
@@ -0,0 +1,147 @@
+package seedu.address.ui;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Manages content generation for a {@code Person}'s detailed view,
+ * including strings for fields, templates, and button labels.
+ */
+public class PersonDetailedViewContentManager {
+    private final Person person;
+
+    /**
+     * Constructs a {@code PersonDetailedViewContentManager} with the specified {@code Person}.
+     * @param person The person whose details are displayed.
+     */
+    public PersonDetailedViewContentManager(Person person) {
+        this.person = person;
+    }
+
+    /**
+     * Gets the full name of the person.
+     * @return A formatted string representing the person's full name.
+     */
+    public String getName() {
+        return person.getName().fullName;
+    }
+
+    /**
+     * Gets the formatted phone number.
+     * @return A formatted string representing the person's phone number.
+     */
+    public String getPhone() {
+        return "+65 " + person.getPhone().value;
+    }
+
+    /**
+     * Gets the formatted address.
+     * @return A formatted string representing the person's address.
+     */
+    public String getAddress() {
+        return "Address: " + person.getAddress().value;
+    }
+
+    /**
+     * Gets the formatted birthday.
+     * @return A formatted string representing the person's birthday.
+     */
+    public String getBirthday() {
+        return "Birthday: " + person.getBirthday().value;
+    }
+
+    /**
+     * Gets the age of the person as a string.
+     * @return A formatted string representing the person's age.
+     */
+    public String getAge() {
+        return "Age: " + person.getAge().value;
+    }
+
+    /**
+     * Gets the email address.
+     * @return A string representing the person's email address.
+     */
+    public String getEmail() {
+        return person.getEmail().value;
+    }
+
+    /**
+     * Gets the formatted paid status.
+     * @return A formatted string representing the person's payment status.
+     */
+    public String getHasPaidStatus() {
+        return "Paid status: " + (person.getHasPaid() ? "Paid" : "Not Paid");
+    }
+
+    /**
+     * Gets the formatted policy renewal frequency.
+     * @return A formatted string representing the person's policy renewal frequency.
+     */
+    public String getFrequency() {
+        return "Policy Renewal Frequency: " + person.getFrequency().value + " month(s)";
+    }
```

Comment:
These methods as well not sure if they are being used or I'm just tripping

Reply:
Believe you are indeed tripping as the methods are used in PersonDetailedView's initialiseView method